src/shared/hooks/use-lara-interactive-api.ts

```diff
@@ -34,6 +34,10 @@ export const useLARAInteractiveAPI = (config: IConfig) => {
     phone.current?.post("interactiveState", state);
   }
 
+  const handleSetHeight = (height: number) => {
+    phone.current?.post("height", height);
+  }
```

Comment:
Is it possible for setHeight to be called before the phone is ready?
If so, should this cache the height and then set it as soon as the phone is ready?

Reply:
I'm pretty sure iframe-phone takes care of that. It queues messages that are posted before the handshake and sends them after the connection is established.

Reply:
Yup that makes sense.